query/src/main/scala/filodb/query/exec/AggrOverRangeVectors.scala

```diff
@@ -57,6 +57,7 @@ final case class AggregateMapReduce(aggrOp: AggregationOperator,
                                     without: Seq[String],
                                     by: Seq[String]) extends RangeVectorTransformer {
   require(without == Nil || by == Nil, "Cannot specify both without and by clause")
+  override def funcParams: Seq[FuncArgs] = Nil
```

Comment:
Shouldn't the default be Nil?  Also please remove the `override` as you are likely just implementing something.  override should be used only if you are truly overriding something

Reply:
Yes moved it to case class